src/pages/en/migrate.md

```diff
@@ -6,6 +6,25 @@ description: How to migrate your project to latest version of Astro.
 
 Until Astro reaches v1.0, we expect to make some breaking changes across minor versions (ex: `v0.1 -> v0.2`). This guide exists to help you migrate to the latest versions of Astro and keep your codebase up-to-date.
 
+## Migrate to v0.25
+
+### A new integration system
+
+The `renderers` config has been replaced by a new, official integration system! When you update to `v0.25.0`, you should see a notice on startup that will tell you how you can update your configuration file to move from the deprecated `renderers` config to the new `integrations` config, complete with an example code snippet for you to copy.
+
```

Comment:
There should also be a note here that the names of renderers have changed.
E.g. `@astrojs/renderer-react` has become `@astrojs/react`, and `@astrojs/renderer-solid` has become `@astrojs/solid-js`. 

This is what took me the longest to figure out when migrating to the per-release.

Reply:
Thanks for the feedback! I'm curious: did you see the message in your console output, and did that help you migrate at all?

Reply:
I don't reccall getting one, no. It took quite a bit of trial and error to get going, but I was also making some other configuration changes at the same time, so that was to be expected.

Reply:
ah, good to know. Updated the docs to be more helpful in case you don't see the message